### PR TITLE
feat(auth-api): Add endpoint for updating claims.

### PR DESCRIPTION
**Pull request by valurefugl**
## What

Add endpoint to ids-api for updating user claims.

## Why

So that the IDS can keep them up to date.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — APPROVED** [review]
LGTM

File: libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts [hunk 1]
```diff
@@ -214,4 +217,30 @@ export class UserIdentitiesService {
...
+          value: c.value,
+          valueType: c.valueType,
+          issuer: c.issuer,
+          originalIssuer: c.originalIssuer,
+        })),
+        { transaction: t },
+      )
+    })
+
+    return this.claimModel.findAll({ where: { subjectId: subjectId } })
```
Comment (saevarma) [inline libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts:244]: As we are writing and reading in the same service call we should add `useMaster: true`, we @eirikurn  saw issue when creating delegations as the query read from read replica which was not updated.

**Review by saevarma — CHANGES_REQUESTED** [review]

File: libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts (hunk 1 above)
Comment (valurefugl) [reply libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts]: Good point, fixed.

File: libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts [hunk 2]
```diff
@@ -241,6 +241,8 @@ export class UserIdentitiesService {
       )
     })
 
-    return this.claimModel.findAll({ where: { subjectId: subjectId } })
+    return this.claimModel.findAll({
+      where: { subjectId: subjectId, useMaster: true },
```
Comment (eirikurn) [inline libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts:245]: `useMaster: true` should be outside the `where` object.

**Review by eirikurn — CHANGES_REQUESTED** [review]

File: libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts (hunk 2 above)
Comment (valurefugl) [reply libs/auth-api-lib/src/lib/user-identities/user-identities.service.ts]: Yikes. Fixed.

**Review by eirikurn — APPROVED** [review]
💯

**Pull request merged**